indico/modules/events/contributions/fields.py

```diff
@@ -65,3 +67,19 @@ def pre_validate(self, form):
                 raise ValueError(_("Author data received"))
             if person.author_type == AuthorType.none and not person.is_speaker:
                 raise ValueError(_("{} has no role").format(person.full_name))
+
+
+class SubContributionPersonListField(ContributionPersonListField):
+    """A field to configure a list of subcontribution persons"""
+
+    def _convert_data(self, data):
+        return list({self._get_subcontribution_person(x) for x in data})
```

Comment:
Do we really need to override `_convert_data`? We are gonna keep the functionality to set submission rights, so the backend should probably handle the data in the same way as in `ContributionPersonListField`.